### PR TITLE
Fixes for the german translation

### PR DESCRIPTION
**Pull request by crulplay**
Some improvements to the german translation. Unfortunately, there is no rel short translation for "Loadout" in German, as it is needed in this context. "Bewaffnung" only refers to the weapons in the loadout. A semantically correct translation would be "Ausrüstungskonfiguration", but I think that would look pretty bad in the UI.

### CODE REVIEW

**Comment by Korben85** [comment]
I wouldn't change the term "Loadout", because as mentioned before there is no good translation for this. So maybe we should leave "Loadout" as it is because almost everybody would understand what it means.

File: app/scripts/dimApp.i18n.js [hunk 1]
```diff
@@ -99,19 +99,19 @@
           General: "Allgemein",
           Postmaster: "Poststelle",
           Equip: "Ausrüsten",
-          Store: "Speichern",
+          Store: "In den Tresor legen",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: Is there a translation that doesn't involve the vault, something generic that means "to hold on to" (e.g. hoard)

File: app/scripts/dimApp.i18n.js [hunk 2]
```diff
@@ -99,19 +99,19 @@
...
-          Super_cooldown: "Super abklingzeit",
-          Grenade_cooldown: "Granaten abklingzeit",
-          Melee_cooldown: "Nahkampf abklingzeit",
+          Super_cooldown: "Super Abklingzeit",
+          Grenade_cooldown: "Granaten Abklingzeit",
+          Melee_cooldown: "Nahkampf Abklingzeit",
           tier_progress: "{{progress}} für {{tier}}",
           About: "Über",
-          'Support DIM': "Unterstützen DIM",
+          'Support DIM': "DIM unterstützen",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: Capitalize both words to maintain current style. Same comment to translations below

File: app/scripts/dimApp.i18n.js (hunk 1 above)
Comment (crulplay) [inline app/scripts/dimApp.i18n.js]: I think "Verstauen" would be possible ( http://www.dict.cc/?s=verstauen )

File: app/scripts/dimApp.i18n.js [hunk 3]
```diff
@@ -121,16 +121,16 @@
           Junk: "Trödel",
           Infuse: "Infundieren",
           notes_placeholder: "Hinzufügen von Anmerkungen zu diesem Artikel",
-          'Create Loadout': "Erstellen Ausladen",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: Yeah I agree with @Korben85, and this is my fault as I found a translation for Loadout, let's just leave loadout as loadout, so Erstellen Loadout, etc

File: app/scripts/dimApp.i18n.js [hunk 4]
```diff
@@ -121,16 +121,16 @@
...
+          loadout_from_equipped: "Aus angelegter Ausrüstung",
+          'Edit Loadout': "Bewaffnung bearbeiten",
+          'Delete Loadout': "Bewaffnung löschen",
+          apply_search_loadout: "Gegenstand = \"{{query}}\"",
+          'Maximize Light': "Licht maximieren",
+          'Item Leveling': "Gegenstand aufwerten",
+          'Gather Engrams': "Engramme sammeln",
           gather_engrams_except_exotics: "Exotics",
-          farming_mode: "Engramme Tresor (Ackerbau)",
+          farming_mode: "Engramme in den Tresor (Sammeln)",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: Engramme zu Tresor?

File: app/scripts/dimApp.i18n.js (hunk 1 above)
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: That works

File: app/scripts/dimApp.i18n.js (hunk 4 above)
Comment (Korben85) [inline app/scripts/dimApp.i18n.js]: Engramme zum Tresor?

**Comment by SunburnedGoose** [comment]
Does this PR need more work?

**Comment by delphiactual** [comment]
Once the changes from the comments are made it can be merged

File: app/scripts/dimApp.i18n.js [hunk 5]
```diff
@@ -121,18 +121,18 @@
           Junk: "Trödel",
           Infuse: "Infundieren",
           notes_placeholder: "Hinzufügen von Anmerkungen zu diesem Artikel",
-          'Create Loadout': "Erstellen Ausladen",
```
Comment (crulplay) [inline app/scripts/dimApp.i18n.js]: I edited the translation according to the reviews.

Comment (SunburnedGoose) [inline app/scripts/dimApp.i18n.js]: Thank you.

**Comment by kyleshay** [comment]
danke schön @crulplay!! ^_^ 

also, special thanks to @delphiactual and @Korben85 for reviewing! :)

**Pull request merged**